ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/_layout/runs/$runIndex/vulnerabilities/index.tsx

```diff
@@ -151,9 +151,16 @@ const renderSubComponent = ({
     <div className='flex flex-col gap-2'>
       <div className='italic text-muted-foreground'>
         <div className='font-semibold'>Description:</div>
-        {row.original.vulnerability.description}
+        <div className='whitespace-pre-line'>
+          {row.original.vulnerability.description}
+        </div>
       </div>
-      <Accordion type='single' collapsible className='ml-4'>
+      <Accordion
+        type='single'
+        collapsible
+        className='ml-4'
+        defaultValue='references'
+      >
```

Comment:
It should probably not be an accordion if it's open by default, just show them if the main thing is open. Should also be a separate commit.